app/views/shared/_profile-accordion-edit.html.erb

```diff
@@ -0,0 +1,120 @@
+<section class="accordion">
+
+  <button class="accordion__control" aria-expanded="true">
+    <h2 class="accordion__title">Personal details</h2>
+  </button>
+  <div class="accordion__content">
+    <%= form.hidden_field :lock_version, :value => contact.lock_version %>
+    <div class="field">
```

Comment:
I thought we were going to make each accordion section a partial so we could reuse them in the create contact form? More trouble than it's worth?

Reply:
Yeah, this ended up being quite tricky as 
1: the styling and tags vary quite a bit between the accordion and the profile edit page, e.g. there are massive h2 elements in the edit profile page that serve as labels, which look out of place in the accordion
2: There are certain contact fields that are being edited alongside the need, that have to be excluded from the accordion